compiler/visq/CMakeLists.txt

```diff
@@ -82,10 +83,35 @@ add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/test
 
 list(APPEND VISQ_TEST_DEPS ${CMAKE_CURRENT_BINARY_DIR}/test)
 
+###
+### Import pics module
+###
+get_target_property(PICS_BIN_PATH pics BINARY_DIR)
+add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/circle
+                   COMMAND ${CMAKE_COMMAND} -E create_symlink
+                   ${PICS_BIN_PATH}/circle ${CMAKE_CURRENT_BINARY_DIR}/circle)
+
+list(APPEND VISQ_TEST_DEPS ${CMAKE_CURRENT_BINARY_DIR}/circle)
+
+###
+### Generate Resources.py
+###
+set(RESOURCE_FILE "${CMAKE_CURRENT_BINARY_DIR}/test/Resources.py")
+
+get_target_property(FP32_MODEL_DIR testDataGenerator BINARY_DIR)
+
+add_custom_command(
+  OUTPUT ${RESOURCE_FILE}
+  COMMAND ${CMAKE_COMMAND} -E echo 'fp32_model_dir=\"${FP32_MODEL_DIR}\"' >> ${RESOURCE_FILE}
+  COMMENT "Generate file to specify resource location"
+)
+
+list(APPEND VISQ_TEST_DEPS ${RESOURCE_FILE})
+
 add_custom_target(visq_unittest ALL DEPENDS ${VISQ_TEST_DEPS})
 
 add_test(
   NAME visq_unittest
-  COMMAND ${PYTHON_EXECUTABLE} -m unittest
+  COMMAND ${NNCC_OVERLAY_DIR}/venv_2_8_0/bin/python -m unittest
```

Comment:
This is necessary to run `visq_unittest` on venv where pydot is installed.

Reply:
is `find_package(PythonInterp 3.8 QUIET)` needed if we change this line?

Reply:
> is find_package(PythonInterp 3.8 QUIET) needed if we change this line?

It is not necessary anymore. Thanks for finding!

> It would be better to add a comment why venv_2_8_0/bin/python is used.

I've added comments. PTAL